### DISCUSSION TITLE
tpv16 Error: Detected Inf/NaN in receiver output. Aborting.

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#576] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/104721358/173101153-c7aead4f-e7c8-4607-9cb1-08d52bf8dca6.png)

**Comment by krenzland** [comment]
Yes, that looks broken. We'll need a tiny bit more detail for a better diagnosis.

**Reply by smallriver666** [reply]
> Yes, that looks broken. We'll need a tiny bit more detail for a better diagnosis.

I'm sorry for not replying to you, I went home from vacation during this time, and I encountered this problem when I implemented this example, but there is no other detailed error message.

**Comment by sebwolf-de** [comment]
What is the content of `tpv16_input_file.txt`?

**Reply by smallriver666** [reply]
> 
![image](https://user-images.githubusercontent.com/104721358/176662120-a6c441bd-9870-43de-b964-d1fb9a70ffd8.png)

**Reply by sebwolf-de** [reply]
Then you should do, what is written there ;-)
`tpv16_input_file.txt` should start with
```
       200         3
       640       260    4.800000E+04    1.950000E+04
       364       117    2.730000E+04    8.775000E+03    1.127366E+03    1.127366E+03
         0         0    0.000000E+00    0.000000E+00    6.000000E+07    2.509795E+07    0.000000E+00    4.182992E-01    0.000000E+00    6.770000E-01    3.730000E-01    4.000000E-01    4.000000E+06    1.000000E+09
```

**Comment by sebwolf-de** [comment, marked as answer]
I guess this problem has been solved, has it? https://github.com/SeisSol/Examples/pull/47 makes the procedure clearer.
